### PR TITLE
Touch content to prevent cleanup

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #9154

### CODE REVIEW

**Comment by daviddavis** [comment]
There was one other place that seems like a `touch()` call might make sense:

https://github.com/pulp/pulp_file/blob/c6300d1e0460ed1273ac31114cccbc4c49683eb7/pulp_file/app/tasks/publishing.py#L67-L69

However, this content must belong to a repo version and therefore it wouldn't be orphaned.

**Review by ipanova — APPROVED** [review]

File: pulp_file/app/serializers.py [hunk 1]
```diff
@@ -38,6 +38,9 @@ def deferred_validate(self, data):
         )
 
         if content.exists():
+            for content in content.iterator():
```
Comment (ipanova) [inline pulp_file/app/serializers.py:41]: +1

**Review by mdellweg — APPROVED** [review]

File: pulp_file/app/serializers.py (hunk 1 above)
Comment (mdellweg) [reply pulp_file/app/serializers.py:41]: We do not expect more than one item here, right? ;)

Comment (ipanova) [reply pulp_file/app/serializers.py:41]: we don't, but i think iterator will not bring any harm..

Comment (daviddavis) [reply pulp_file/app/serializers.py:41]: I like `content.get().touch()`. Will update.

**Pull request merged**